extension/lib/doofinder/Client.js

```diff
@@ -4,19 +4,22 @@ const { promisify } = require('util')
 class Client {
   /**
    * @param {Object} config
-   * @param {string} authHeader
    */
   constructor ({config, tracedRequest, log}) {
```

Comment:
Why do we need a `tracedRequest` ? It will pollute jaeger, datadog, etc with obsolete logs and metrics

Reply:
Thats what we do making API request within PS extensions.
https://github.com/shopgate-professional-services/ext-bolt-checkout/blob/d6ba8828fffacfcb84fbb6e481646c22b939da77/extension/utilities/BoltApi.js#L10
Should be discussed within the team.